webgui/scripts/codecompass/view/fileManager.js

```diff
@@ -237,6 +239,46 @@ function (on, query, style, topic, declare, Memory, Observable,
       path.unshift('project');
 
       this.set('path', path);
+      this.set('filePath', fileInfo.path);
+    },
+
+    _setFilePathAttr : function (path) {
+      this._filePath = path;
+      this._setFilePath();
+    },
+
+    /*
+     * Calculate the actual width of an element
+     */
+    getWidthOfAnElement : function(node){
+      if(node){
+        var computedStyle = style.getComputedStyle(node);
+        return geom.getMarginBox(node, computedStyle).w;
+      }else
+        return 0;
+    },
+
+    _setFilePath : function () {
```

Comment:
The path to set should be passed as a parameter to this function. `this._filePath` is not needed at all.